yarn.lock

```diff
@@ -1972,6 +1972,14 @@
     "@griffel/react" "^1.0.0"
     tslib "^2.1.0"
 
+"@fluentui/react-icons@^2.0.239":
```

Comment:
looks like you did miss some packages, this dupe cannot happen, please adjust your changes